zcash_client_sqlite/src/testing/pool.rs

```diff
@@ -1956,8 +1961,10 @@ pub(crate) fn invalid_chain_cache_disconnected<T: ShieldedPoolTester>() {
         disconnect_height,
         BlockHash([1; 32]),
         &dfvk,
-        AddressType::DefaultExternal,
-        NonNegativeAmount::const_from_u64(8),
+        &[FakeCompactOutput {
+            address_type: AddressType::DefaultExternal,
+            value: NonNegativeAmount::const_from_u64(8),
+        }],
```

Comment:
Ditto:
```suggestion
        &[FakeCompactOutput::new(
            AddressType::DefaultExternal,
            NonNegativeAmount::const_from_u64(8),
        )],
```

EDIT: Looks like this was fixed in the second commit. I'd prefer it to be done in the first commit if you're making other changes to that commit.